dev_guide/templates.adoc

```diff
@@ -82,13 +89,27 @@ $ oc create -f <filename> -n <project>
 The template is now available to be selected for a configuration using the web
 console or the CLI.
 
-=== Generating a Configuration
-Generate a configuration with the following command:
+=== Generating a list of resources
+Process the template and return the list of resources to standard output:
 
 ----
 $ oc process -f <filename>
 ----
 
+You can also process the uploaded template by calling the `process` command using:
+
+----
+$ oc process uploaded-template-name
+----
+
+The process command also takes a list of templates you can process to a list of
+resources. In that case, every template will be processed and the resulting list
+of resources will contain objects from all templates passed to a process command:
```

Comment:
hmm.  you can run oc process -f file1.json -f file2.json

but only flie2 gets processed in that case.

seems like tihs is a path that should work..having to cat to do it seems unnatural.  but definitely low priority (processing multiple templates in a single invocation doesn't strike me as a strong use case, especially given how easy it would be to just script multiple calls)